### DISCUSSION TITLE
Can't see the attached files in submissions.

### DISCUSSION THREAD

**Discussion by glittery11** [team-forms/community#28] [Q&A]
I put a file in the attachment field and submit it, and it submits fine, but I can't see it on the submission page or in the PDF file.

Approvers currently can't even see if a respondent has attached a file, only admins can see it in the submission list. Is this an error or an unimplemented feature?

![image](https://github.com/team-forms/community/assets/88357418/847edfdb-954f-454d-9fed-8fcfe3f22c3c)

![image](https://github.com/team-forms/community/assets/88357418/6adcbfdc-4447-4a7f-b7a9-21156539a370)

**Comment by Erin-Dwyer** [comment]
@glittery11 attached files should appear in both the generated pdf and in the form as shown below. I wonder if there as some issue with the specific submission shown? Have you since run additional test and been able to reproduce the issue?

<img width="1407" alt="image" src="https://github.com/team-forms/community/assets/154988548/7d1e9648-77c8-4f53-b130-34c1419d607d">

<img width="866" alt="image" src="https://github.com/team-forms/community/assets/154988548/c050ba5c-b1fa-4120-8cd6-aab89b4acec3">
